bliss/simulator/prior.py

```diff
@@ -206,10 +206,31 @@ def _load_color_distribution(self, sdss_fields):
             run = field_params["run"]
             camcol = field_params["camcol"]
             fields = field_params["fields"]
+            # load project SDSS dir
+            sdss_path = Path(self.sdss)
+
+            # Set photoField file Path
+            camcol_path = sdss_path.joinpath(str(run), str(camcol))
+            pf_file = f"photoField-{run:06d}-{camcol:d}.fits"
+            pf_path = camcol_path.joinpath(pf_file)
+            if not Path(pf_path).exists():
+                self.downloader.download_pf()
+            msg = (
+                f"{pf_path} does not exist. "
+                + "Make sure data files are available for specified fields."
+            )
+            assert Path(pf_path).exists(), msg
```

Comment:
Instead of manually loading the files here, it would be better to just instantiate an SDSS object (or pass one as a parameter).

Reply:
I figured I would just load the frames individually since the instantiation of an SDSS object results in some additional overhead (mostly in `read_frame_for_band`) that I thought could be avoided since we probably want to keep everything during generation as cut and dry as possible. It definitely would have been a lot easier to just instantiate an SDSS object, but I thought it was worth the effort.

Reply:
You're reading the frames anyway though in `_compute_flux_ratio`. Unless there is a frame with no valid objects, I think you're doing the same amount of work either way, and imo just using the SDSS object is much cleaner than duplicating all of its functionality here (and also makes maintaining it much easier).

Reply:
Yeah it would make sense if we're planning on relying on the SDSS object going forward and for maintenance. Just updated to do this in a new commit.